src/main/java/hudson/plugins/ec2/EC2Cloud.java

```diff
@@ -642,6 +642,8 @@ else if (jenkinsInstance.isTerminating()) {
     private PlannedNode createPlannedNode(final SlaveTemplate t, final EC2AbstractSlave slave) {
         return new PlannedNode(t.getDisplayName(),
                 Computer.threadPoolForRemoting.submit(new Callable<Node>() {
+                    int retryCount     = 0;
+                    int DESCRIBE_LIMIT = 1;
```

Comment:
There probably should be more than 1 retry and it should be `final`

Reply:
Thanks for catching that, just fixed it and increased the count to 2